www/jsonrpc/server.go

```diff
@@ -50,32 +48,18 @@ func (s *Server) StartServer(grpcServer string) error {
 
 	s.grpcClient = grpcConn
 
-	blockchain := pactus.NewBlockchainClient(grpcConn)
-	network := pactus.NewNetworkClient(grpcConn)
-	transaction := pactus.NewTransactionClient(grpcConn)
-	wallet := pactus.NewWalletClient(grpcConn)
+	blockchainService := pactus.RegisterBlockchainJsonRpcService(grpcConn)
+	networkService := pactus.RegisterNetworkJsonRpcService(grpcConn)
+	transactionService := pactus.RegisterTransactionJsonRpcService(grpcConn)
+	walletService := pactus.RegisterWalletJsonRpcService(grpcConn)
 
-	blockchainService := pactus.NewBlockchainJsonRpcService(blockchain)
-	networkService := pactus.NewNetworkJsonRpcService(network)
-	transactionService := pactus.NewTransactionJsonRpcService(transaction)
-	walletService := pactus.NewWalletJsonRpcService(wallet)
-
-	jgw := jrpc.NewServer()
-	jgw.RegisterServices(&blockchainService, &networkService, &transactionService, &walletService)
+	server := jrpc.NewServer()
+	server.RegisterServices(blockchainService, networkService, transactionService, walletService)
```

Comment:
Can we merged this line into above register functions. We may pass  `server` to the `RegisterBlockchainJsonRpcService`?

Reply:
Registers create handers for server and server add this handlers to http.ServerMux